app/views/layouts/frontend.html.erb

```diff
@@ -10,7 +10,7 @@
       <li><a href="/news-and-communications">News and communications</a></li>
     </ul>
   </nav>
-  <main id="content" role="main" class="whitehall-content">
+  <main id="content" role="main" class="whitehall-content"<%= " lang=#{locale.to_s}" unless locale == :en %>>
     <div id="page" class="<%= content_for?(:page_class) ? yield(:page_class) : '' %>" >
```

Comment:
I noticed this differs slightly from government-frontend which uses I18n.locale but I'm not clear on why that would be better and I'm sure you've tested this is inserting the correct codes so probably no change to make here.

Reply:
Thanks for sharing - I've changed it to use `I18n.locale`. My thinking is that `I18n.locale` is only about the translation, whereas `locale` could be hijacked to something more general.